### DISCUSSION TITLE
GSoC 2024 Applications are now open!

### DISCUSSION THREAD

**Discussion by pradeeban** [uaanchorage/GSoC#6] [Announcements]
Please make sure to submit your applications on time.

March 18 - 18:00 UTC

    GSoC contributor application period begins

April 2 - 18:00 UTC

    GSoC contributor application deadline

Please note that you can submit your proposal and continue to edit it. I recommend that, rather than waiting for the deadline to submit.

Oftentimes, the GSoC page gets unresponsive during the last minutes of the submission deadline as hundreds of applicants try to submit or access the page at once! Similarly, things outside your control can happen (such as power failure, internet disruption, or personal issues). Google is very strict about the application deadline. Therefore, it is important you submit well in advance, as organizations cannot do anything about a missed deadline!

We are happy to see the enthusiasm and interactions from the applicants. We are looking forward to working with you this summer.

**Comment by bisratberhanu** [comment]
Hi, my name is Bisrat Berhanu,  a 4th year electrical and computer Engineering student at Addis Ababa University. I would like to be a contributor to the coastline project. I am eager to learn machine learning and open source contributing through this project. I am proficient in python, data structures and algorithms. I also have basic knowledge of machine learning. If anyone would like to tell me more about, what type of proposal should be submit, it is very much appreciated.
